k8_cortx_cloud/upgrade-cortx-cloud.sh

```diff
@@ -67,6 +67,16 @@ function parse_solution() {
   "${DIR}/parse_scripts/parse_yaml.sh" "${SOLUTION_FILE}" "$1"
 }
 
+function validate_upgrade_images() {
+    image="$1"
+    if docker pull "${image}" > /dev/null; then
+        echo "Image ${image} pulled successfully"
```

Comment:
Lets use word "download" instead of "pull"
Success: Image download...
Error: Image download, provide valid image...

Reply:
done